app/code/community/Drip/Connect/Helper/Data.php

```diff
@@ -132,6 +132,22 @@ public function isCustomerExists($email, $websiteId = null)
         return (bool) $customer->getId();
     }
 
+    /**
+     * find customer by email address
+     *
+     * @return Mage_Customer_Model_Customer $customer
+     */
+    public function getCustomerByEmail($email, $websiteId = null)
```

Comment:
Instead of passing the websiteId in here and accepting null, could we pass the config object in here, and add a `getWebsiteId` method to the configuration class which determines the website id from the store view id? Also, I'm trying to always explicitly pass config objects around and never depend on the global store id and such except at the boundaries where we initialize the config class.